api/src/test/java/org/openmrs/util/databasechange/Database1_9_7UpgradeIT.java

```diff
@@ -446,4 +446,15 @@ public void shouldSetValuesToNullIfUnitsOrFrequencyBlank() throws Exception {
 		assertThat(drug_orders, containsInAnyOrder(row("order_id", "6", "dose_units", null, "frequency", null), row(
 		    "order_id", "7", "dose_units", null, "frequency", null)));
 	}
+
+	@Test
+	public void shouldUpgradeLiquibase() {
+		try {
+			upgradeTestUtil.upgrade("liquibase-master.xml");
+		} catch (Exception e) {
+			e.printStackTrace();
```

Comment:
Also do not catch the exception. It's better to fail with an exception and not Assert.fail()